pylzy/lzy/api/v1/provisioning.py

```diff
@@ -1,6 +1,66 @@
-from dataclasses import dataclass
+import dataclasses
+
 from enum import Enum
-from typing import Optional
+from typing import (
+    Any as typingAny,
```

Comment:
nit: Mb not rename Any, just use it as typing.Any?

Reply:
I thought about it, but it will complicate the code, because I'm using my Any as a type, but also as a value. typing.Any, from the other hand - can't be instantiated :(
But I left a possibility for the user to use typing.Any here, just they's mypy tests will go crazy/